assets/helpers/abTests/helpers.js

```diff
@@ -0,0 +1,22 @@
+// @flow
```

Comment:
Is it possible to give a more meaningful name to this file? Why don't we move this functions inside the `abTest.js` helper? If you would like to keep this as a separate file, an alternative name could be `participations.js`.

Reply:
I'll move it into abTest if you like, my thought was that that file is getting quite large and it might be good to separate out the mechanics of setting up AB tests from helper methods which are used elsewhere in the code base.